packages/trader/src/Modules/Trading/Containers/Multiplier/multiplier-amount-modal.jsx

```diff
@@ -54,11 +58,42 @@ export default connect(({ client, modules, ui }) => ({
     disableApp: ui.disableApp,
 }))(MultiplierAmountModal);
 
-const TradeParamsMobile = ({ amount, currency, toggleModal }) => {
+const TradeParamsMobile = ({ amount, currency, toggleModal, trade_store }) => {
     const [stake_value, setStakeValue] = React.useState(amount);
+    const [commission, setCommission] = React.useState(null);
+    const [stop_out, setStopOut] = React.useState(null);
+    const stake_ref = React.useRef(amount);
+    const is_mounted = useIsMounted();
+
+    React.useEffect(() => {
+        if (stake_value === amount) return undefined;
```

Comment:
```suggestion
        if (stake_value === amount) return;
```

Reply:
We'll need to keep this `undefined` as we're also returning `dispose` in the arrow function for `cleanup`. Using the suggested change without `undefined` will violate `consistent-return` `eslint` rule.